### ISSUE TITLE
Want to automatically remove the mobile notch region similar to Godot-3.2.2

### ISSUE THREAD

**Issue by abhiagnihotri01** [godotengine/godot#49356]
**Godot version:** Godot-3.3.2

**OS/device including version:** Mi Pro 7 (Android)

**Issue Description:**
In godot-3.3.2, there is no option for automatically remove the notch region so that notch removal from mobile become similar to godot-3.2.2.
How to remove mobile notch region automatically without using  OS.get_window_safe_area()?

**Comment by Chaosus** [comment]
What is notch region?

**Comment by Zireael07** [comment]
Notch, I believe, is what they call the weirdly shaped bits/edges on a mobile screen that's not square.

https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.androidauthority.com%2Fwho-owns-the-notch-845086%2F&psig=AOvVaw1pgsKhuF69jlkG_IvMTF58&ust=1623070780811000&source=images&cd=vfe&ved=2ahUKEwjwk_r7h4PxAhWKsyoKHVanD7wQr4kDegUIARDAAQ

**Comment by abhiagnihotri01** [comment]
Notch is front camera region which comes over the screen.

**Comment by Calinou** [comment]
For the record, [this question was cross-posted on the Godot Q&A](https://godotengine.org/qa/106641/how-backout-window_safe_area-changes-and-make-similar-godot).

The solution is to edit the `AndroidManifest.xml` file with:

```
android:theme="@android:style/Theme.Black.NoTitleBar.Fullscreen"
```

**Comment by madmiraal** [comment]
Fixed with #61472.

Disabling the Android export setting `screen/immersive_mode` now works.
